modules/pcd/test/pcd-loader.spec.js

```diff
@@ -30,12 +30,32 @@ test('PCDLoader#parseBinary', t => {
   const data = parseFileSync(PCD_BINARY, PCDLoader);
   validateLoadedData(t, data);
 
-  // Check internal loader data
-  t.ok(data.loaderData.header, 'Original header was found');
-
   t.equal(data.mode, 0, 'mode is POINTS (0)');
   t.notOk(data.indices, 'INDICES attribute was not preset');
   t.equal(getGLTFAttribute(data, 'POSITION').value.length, 179250, 'POSITION attribute was found');
 
   t.end();
 });
+
+test('PCDLoader#parseBinaryAsync', t => {
+  if (typeof Worker === 'undefined') {
+    t.comment('Worker is not usable in non-browser environments');
+    t.end();
+    return;
+  }
+
+  // Once binary is transferred to worker it cannot be read from the main thread
+  // Duplicate it here to avoid breaking other tests
```

Comment:
Couldn't it be transferred back when worker is done? Would be nice to avoid this little "gotcha", surely some user will be "burnt" by it.

Reply:
- No. We can transfer the arraybuffer back as another variable, but the old one cannot be restored.
- The worker test is executed in parallel to other tests. While the worker is processing any other test that tries to access this data will fail.